pkg/network/node/egress_network_policy.go

```diff
@@ -71,14 +71,16 @@ func (plugin *OsdnNode) handleEgressNetworkPolicy(policy *osdnv1.EgressNetworkPo
 	plugin.egressPoliciesLock.Lock()
 	defer plugin.egressPoliciesLock.Unlock()
 
+	var oldPolicySave osdnv1.EgressNetworkPolicy
 	policies := plugin.egressPolicies[vnid]
 	for i, oldPolicy := range policies {
 		if oldPolicy.UID == policy.UID {
 			policies = append(policies[:i], policies[i+1:]...)
+			oldPolicySave = oldPolicy
```

Comment:
@JacobTanenbaum I was thinking, why don't we just put the `plugin.egressDNS.Delete(oldPolicy)` here?

If we can't find the old policy in the list, then oldPolicySave will be empty, and the Delete() will just do nothing anyway.

Reply:
done